glossary.md

```diff
@@ -94,6 +94,16 @@ instance: Imagine a satelite (`observing system`) that is very important to a se
 latency (`gaps`. Those would be linked by a thick (high `criticality`) red (low 
 `performance`) line. 
 
+
+```mermaid
+flowchart LR
+    A[Observing Systems]--> |supports| B[Data Products]
+    A --> |self| A
+    B --> |supports| C(Applications)
+    C --> C
+    C --> |supports| D[Societal Benefit Areas]
+```
```

Comment:
@hazelshapiro you can change or request changes here :)

Reply:
Needs a B --> B link and then it'll be final.